File: algorithms/bit-manipulation/unary-coding/csharp/UnaryCoding.cs
using System;

class UnaryCoding
{
    static string UnaryEncode(int n)
    {
        return new string('1', n) + "0";
    }

    static void Main(string[] args)
    {
        Console.WriteLine("Unary encoding of 0: " + UnaryEncode(0));
        Console.WriteLine("Unary encoding of 3: " + UnaryEncode(3));
        Console.WriteLine("Unary encoding of 5: " + UnaryEncode(5));
    }
}

File: algorithms/bit-manipulation/unary-coding/scala/UnaryCoding.scala
object UnaryCoding {
  def unaryEncode(n: Int): String = {
    "1" * n + "0"
  }

  def main(args: Array[String]): Unit = {
    println(s"Unary encoding of 0: ${unaryEncode(0)}")
    println(s"Unary encoding of 3: ${unaryEncode(3)}")
    println(s"Unary encoding of 5: ${unaryEncode(5)}")
  }
}
